app/config/collections.php

```diff
@@ -2764,6 +2764,45 @@
             ],
         ]
     ],
+    'cache' => [
+        '$collection' => Database::METADATA,
+        '$id' => 'cache',
+        'name' => 'Cache',
+        'attributes' => [
+            [
+                '$id' => 'dateAccessed',
+                'type' => Database::VAR_INTEGER,
+                'format' => '',
+                'size' => 0,
+                'signed' => false,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
```

Comment:
Indentation seems wrong here, we can run the coding standards fixer.

Reply:
Ok

Reply:
fixed